packages/dev/core/src/Cameras/camera.ts

```diff
@@ -970,6 +1027,12 @@ export class Camera extends Node {
         return this._transformMatrix;
     }
 
+    private _computeObliqueDistance(): number {
+        const arcRotateCamera = this as unknown as ArcRotateCamera;
+        const targetCamera = this as unknown as TargetCamera;
+        return (arcRotateCamera.radius || (targetCamera.target ? Vector3.Distance(this.position, targetCamera.target) : this.position.length())) + this.oblique!.offset;

```

Comment:
If you pass in the oblique offset, you won't need the `!`

Reply:
Changed